README.md

```diff
@@ -20,6 +20,14 @@ Supported platforms:
 
 ## Usage
 
-This repository contains C-language examples that link to the native library targets and perform basic rendering and computation. You can test them with `make`.
+This repository contains C-language examples that link to the native library targets and perform basic rendering and computation.
+To run the C triangle example, install a C compiler + glfw 3, then run these commands at the root of the repo:
+```
+rustup toolchain install nightly
+cargo install cbindgen
+make examples-native
```

Comment:
Sorry about the changes underneath, but there is no longer "examples-native".
Please update to latest folder structure, and we'll proceed.

Reply:
`make examples-native` still works, I don't understand what your requesting?